src/backend/executor/process.ts

```diff
@@ -19,6 +20,11 @@ export enum ProcessType {
     other = 'Other process',
 }
 
+// Expand an initial '~' component in the given path if there is any, otherwise returns the file path without changes.
+function expandUser(filePath: string) {
+    return os.homedir ? filePath.replace(/^~(?=$|\/|\\)/, os.homedir) : filePath;
```

Comment:
`os.homedir` is a function, not a string.
Technically it still works because .replace() accepts a `(...) => string` as second argument, but it'll fail on systems with no homedir.